tasks/release.py

```diff
@@ -151,15 +151,16 @@ def update_dca_changelog(ctx, new_version, agent_version):
     current_branchoff = f"dca-{new_version_int[0]}.{new_version_int[1]}.X"
     # generate the new changelog. Specifying branch in case this is run outside the release branch that contains the tag.
     ctx.run(
-        f"reno --rel-notes-dir releasenotes-dca report             --ignore-cache             --branch {current_branchoff}             --version dca-{new_version}             --no-show-source > /tmp/new_changelog-dca.rst"
+        f"reno --rel-notes-dir releasenotes-dca report --ignore-cache --branch {current_branchoff} --version dca-{new_version} --no-show-source > /tmp/new_changelog-dca.rst"
     )
 
     # reseting git
     ctx.run("git reset --hard HEAD")
 
     # mac's `sed` has a different syntax for the "-i" paramter
-    sed_i_arg = "-i"
-    if sys.platform == 'darwin':
+    if ctx.run("sed --version", hide='both'):
```

Comment:
How does this work? What are you testing here?

Reply:
It would be great if you can add a comment about this

Reply:
Done!